README.md

```diff
@@ -40,21 +40,30 @@ to perform certain actions on behalf of the user.
 ### client
 Contains client for working with NeoFS.
 ```go
-c, _ := client.New(
-    client.WithAddress("localhost:40005"), // endpoint address
-    client.WithDefaultPrivateKey(key),     // private key for request signing
-    client.WithNeoFSErrorHandling(),       // enable erroneous status parsing
-    client.WithTLSConfig(&tls.Config{}))   // custom TLS configuration
+var prmInit client.PrmInit
+prmInit.SetDefaultPrivateKey(key) // private key for request signing
+prmInit.ResolveNeoFSFailures() // enable erroneous status parsing
+
+var c client.Client
+c.Init(prmInit)
+
+var prmDial client.PrmDial
+prmDial.SetServerURI("grpcs://localhost:40005") // endpoint address
+
+_ = c.Dial(prmDial)
```

Comment:
Let's not ignore errors in examples, we already handle it below.

Reply:
Done similar